worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/MarkupHeadersAndBody.java

```diff
@@ -65,6 +69,7 @@ public MarkupHeadersAndBody(
 
         // Set up the regular expression that matches the condensed header
         final String onDateSomebodyWroteRegEx = regexSetup(condensedHeaderMultilangMappings);
+        this.multilangHeadersElements = fillInList(condensedHeaderMultilangMappings);
```

Comment:
`fillInList` is not a good method name.  This method creates a list - it doesn't fill one it.  And a list of what?